q2_vsearch/plugin_setup.py

```diff
@@ -325,12 +313,9 @@
     },
     name='Join paired-end reads.',
     description=('Join paired-end sequence reads using vsearch\'s '
```

Comment:
I think we should eventually rename this action to `merge_pairs` (and update the description to "Merge paired-end sequence reads..."), as this is the command we're calling (`--fastq_mergepairs`), and merging reads is not the same as joining reads. This may cause confusion, as there is a `--fastq_join` command which will stitch each read together end-to-end with a padding string, usually Ns. Probably do not need to do this now, but I just wanted to mention it while I was here. :-)

Reply:
Want to file an issue for that?